cmd/kubectl-gadget/utils/trace.go

```diff
@@ -406,7 +376,7 @@ func SetTraceOperation(traceID string, operation string) error {
 
 // waitForTraceState loops over all trace whom ID is given as parameter
 // waiting until they are in the expected state.
-func waitForTraceState(traceID string, expectedState string) (gadgetv1alpha1.TraceList, error) {
+func waitForTraceState(traceID string, expectedState string) (*gadgetv1alpha1.TraceList, error) {
```

Comment:
I like that you took advantage of this PR to start using the pointer. Great move!